docs/src/main/paradox/kubernetes-deployment/deploying.md

```diff
@@ -0,0 +1,124 @@
+# Deploying
```

Comment:
Shall we have a separate section about Minikube, as we have in the readme https://github.com/akka/akka-sample-cluster-kubernetes-scala/blob/master/README.md ?

Reply:
👍